massdriver-application/aws_identity.tf

```diff
@@ -6,9 +6,9 @@ locals {
   }
 
   aws_service                      = local.aws_service_map[var.service]
-  aws_eks_oidc_short               = local.is_aws && local.is_kubernetes ? replace(var.kubernetes.cluster_artifact.data.infrastructure.oidc_issuer_url, "https://", "") : null
-  aws_eks_oidc_federated_principal = local.is_aws && local.is_kubernetes ? "arn:aws:iam::${data.mdxc_cloud.current.id}:oidc-provider/${local.aws_eks_oidc_short}" : null
-  aws_eks_assume_role_conditional  = local.is_aws && local.is_kubernetes ? "system:serviceaccount:${var.kubernetes.namespace}:${var.name}" : null
+  aws_eks_oidc_short               = local.is_aws && local.is_kubernetes ? replace(var.kubernetes.cluster_artifact.data.infrastructure.oidc_issuer_url, "https://", "") : ""
```

Comment:
for what it's worth, here's a thread on when I did this. :)
https://github.com/massdriver-cloud/terraform-modules/pull/25#discussion_r956888413

Reply:
It ended up breaking in another scenario. Let terraform have it’s nullish ways I suppose. It was evaluation the untrue side of the ternary when true. Very bizarre.